classes/install/Upgrade.inc.php

```diff
@@ -1577,6 +1582,10 @@ function convertCommentsToEditor() {
 			$comments_to_ed = PKPString::stripUnsafeHtml($row['comments_to_ed']);
 			
 			if ($comments_to_ed != ""){
+				
+				$user_id = $row['user_id'];
+				if (!$user_id) $user_id = 1;
```

Comment:
actually to set the user_id to 1 or the manager is just a fallback. At least in our case we can get the correct user_id every time.